### ISSUE TITLE
Finetuning on my dataset

### ISSUE THREAD

**Issue by kk1153** [meijieru/crnn.pytorch#70]
I have small dataset of 660 cropped text images. I used the instructions mentioned for training. I changed the display, save and val interval according to my dataset ie. 11 and number of iterations as 750.

I am new to pytorch. Can you suggest me what learning rate shall i use for fine tuning and what was the learning rate that you used? How can i better do the finetuning here like for freezing weights at some layers in pytorch? How and which layers shall i use higher learning rate in pytorch? Which synthetic image generator did you use? Are your results similar to the original CRNN?

**Comment by meijieru** [comment]
This issue should be asked at PyTorch's forum, so I close it.

**Comment by kk1153** [comment]
Ok. Can you please tell what learning rate did you use? What specific image generator did you use and how many images you used? How many epoch did you train on?

**Comment by northeastsquare** [comment]
@kk1153   How about this:  https://github.com/JarveeLee/SynthText_Chinese_version/blob/master/gen.py
